var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -158,6 +158,12 @@ class Paraview(CMakePackage, CudaPackage):
     # Broken downstream FindMPI
     patch('vtkm-findmpi-downstream.patch', when='@5.9.0')
 
+    # Freetype@2.10.3 no longer exports FT_CALLBACK_DEF, this
+    # patch replaces FT_CALLBACK_DEF with simple extern "C"
+    # See https://gitlab.kitware.com/vtk/vtk/-/issues/18033
+    patch('pv-vtk-freetype-2.10.3-replace-FT_CALLBACK_DEF.patch',
```

Comment:
This will also need a VTK version constrait since VTK has removed the `FT_CALLBACK_DEF` in the upstream.

Also, in the VTK upstream there is no `extern "C"` required, I think fontconfig handles this already in their header. It isn't wrong, just different from what is in upstream VTK, I am not sure if it is worth changing at this point since this has already been waiting so long.

Reply:
This was fixed in `vtk@9.0.2:` and in `paraview@5.9:`

Reply:
Note, VTK currently has a dependency check so the patch is never applied. I wonder if it is better to just require a newer freetype for newer ParaView rather than try to patch this.

Depending on whether or not older freetype is absolutely required for some technical reason on a target system, I would prefer to just make this a dependency constraint and remove the patch from both VTK and ParaView packages.

Reply:
Sorry for the delayed response.  Missed the emails  and tied up with other projects.  I think your suggestion re requiring newer freetype for recent paraviews (dependency constraint) is preferable to the patches.